packages/aws-cdk-lib/aws-ec2/test/integ.launch-template-capacity-reservation.lit.ts

```diff
@@ -0,0 +1,52 @@
+import * as cdk from '../../core';
+import * as ec2 from '../lib';
+
+class LaunchTemplateCapacityReservationStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    /// !show
+    // LaunchTemplate with capacity reservation specification
+    new ec2.LaunchTemplate(this, 'LaunchTemplateWithCapacityReservation', {
+      machineImage: ec2.MachineImage.latestAmazonLinux2023(),
+      capacityReservationSpecification: {
+        capacityReservationPreference: ec2.CapacityReservationPreference.OPEN,
+        capacityReservationTarget: {
+          capacityReservationId: 'cr-1234567890abcdef0',
```

Comment:
I'm going code blind from all the times that the words `capacityReservation` shows up here. Why is this not:

```ts
capacityReservation: CapacityReservation.fromId(this, 'Reservation', 'cr-1234567890abcdef0'),
capacityReservationPreference: ec2.CapacityReservationPreference.OPEN,
```